packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -334,7 +334,7 @@ const OTPInput = ({
             hideLabelText={true}
             id={`${inputId}-${index}`}
             textAlign="center"
-            ref={ref}
+            ref={ref as never}
```

Comment:
why never?

Reply:
Because I NEVER want to mess with absolute mess of React's ref types.